.github/workflows/build.yml

```diff
@@ -21,60 +21,59 @@ jobs:
     name: Linux
     runs-on: ubuntu-latest
```

Comment:
Perhaps we should fix the Ubuntu version so that the apt repository doesn't get out of sync?

Reply:
We could, but then instead of going out of sync, it will go stale instead :)

Speaking of, is there any reason you’re installing PostgreSQL from the official repository here, instead of just using the PostgreSQL version that comes with the Ubuntu GitHub Actions image? 

Maybe the best path to prevent the two versions from going out of sync is to remove the custom PostgreSQL installation altogether?

Reply:
I have no memory of why. Let's rip it out! 😄